### PR TITLE
Remove go-tus dependency replace as PR has been merge to master

### PR DESCRIPTION
**Pull request by DnlLrssn**
**Checklist**

- [ ] tests added
- [ ] commits conform to the [Commit message format](./CONTRIBUTING.md#commit)
- [ ] documentation updated

**Squash commit message**

andrewmostello's branch has been merged to go-tus master, remove go.mod replace of this dependency and update to latest go-tus master.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -27,6 +25,7 @@ require (
 	github.com/spf13/pflag v1.0.5 // indirect
 	github.com/stretchr/testify v1.6.1
 	github.com/tus/tusd v1.3.0 // indirect
+	golang.org/x/net v0.0.0-20200707034311-ab3426394381 // indirect
```
Comment (atluq) [inline go.mod:28]: Why is this dep showing up here?

`~/go/src/github.com/qlik-oss/gopherciser dependencies/remove-tus-replace
❯ go mod why golang.org/x/net
go: finding module for package golang.org/x/net
# golang.org/x/net
(main module does not need package golang.org/x/net)`

Comment (DnlLrssn) [reply go.mod:28]: not sure that `why` show indirect dependencies. it was added by `go get` and it's not cleaned up by running `go mod tidy; go mod verify` so I will assume go-tus or one of it's dependencies uses it.

Comment (DnlLrssn) [reply go.mod:28]: hmm no, seems to be gopherciser, not sure then 
```
➜  gopherciser git:(dependencies/remove-tus-replace) ✗ go mod graph | grep golang.org/x/net@v0.0.0-20200707034311-ab3426394381
github.com/qlik-oss/gopherciser golang.org/x/net@v0.0.0-20200707034311-ab3426394381
golang.org/x/net@v0.0.0-20200707034311-ab3426394381 golang.org/x/text@v0.3.0
golang.org/x/net@v0.0.0-20200707034311-ab3426394381 golang.org/x/sys@v0.0.0-20200323222414-85ca7c5b95cd
golang.org/x/net@v0.0.0-20200707034311-ab3426394381 golang.org/x/crypto@v0.0.0-20200622213623-75b288015ac9

```

Comment (atluq) [reply go.mod:28]: I guess the tooling should solve this, but after removed manually and, `go mod verify`, and `make verify` did not complain. Interesting...

Comment (DnlLrssn) [reply go.mod:28]: well it was the tooling which added it in the first place though...
and yes manual removal and even with re-running `go mod tidy; go mod verify` it's still removed.. odd indeed

Comment (DnlLrssn) [reply go.mod]: pushed a removal

**Pull request merged**